x/ccv/provider/keeper/msg_server.go

```diff
@@ -474,6 +474,16 @@ func (k msgServer) UpdateConsumer(goCtx context.Context, msg *types.MsgUpdateCon
 		return &resp, errorsmod.Wrapf(ccvtypes.ErrInvalidConsumerState, "cannot get consumer chain ID: %s", err.Error())
 	}
 
+	if strings.TrimSpace(msg.NewChainId) != "" && msg.NewChainId != chainId {
+		if k.IsConsumerPrelaunched(ctx, consumerId) {
+			chainId = msg.NewChainId
```

Comment:
Do the same if the owner address is updated (in the event attributes).

Reply:
Not sure what you mean by this.
Are you saying we should do `ownerAddress  = msg.NewOwnerAddress` [here](https://github.com/cosmos/interchain-security/blob/a7fa6e68d969ff31e9b6524803721993d8fbf243/x/ccv/provider/keeper/msg_server.go#L502)?
If we were to do this, we would mess up the following [check](https://github.com/cosmos/interchain-security/blob/a7fa6e68d969ff31e9b6524803721993d8fbf243/x/ccv/provider/keeper/msg_server.go#L558) and allow for Top N chains to be created outside governance.